subsys/net/ip/net_context.c

```diff
@@ -689,6 +689,45 @@ int net_context_put(struct net_context *context)
 	return 0;
 }
 
+static int net_context_bind_default(struct net_context *context,
+				    const struct sockaddr *dest_addr)
+{
+	struct net_if *iface = net_if_get_default();
+
+	if (!iface) {
+		NET_ERR("Cannot bind to %s",
+			net_sprint_ipv6_addr(&addr6->sin6_addr));
+
+		return -EADDRNOTAVAIL;
+	}
+
+	if (dest_addr->family == AF_INET6) {
```

Comment:
I think we should put
```
#if defined(CONFIG_NET_IPV{6|4})
...
#endif /* ... */
```
 
checks around the family checkings, so similar way as what is done in other functions.